src/Components/Components/src/RenderTree/RenderTreeBuilder.cs

```diff
@@ -224,23 +224,6 @@ public void AddAttribute(int sequence, string name, Action value)
             AddAttribute(sequence, name, (MulticastDelegate)value);
         }
 
-        /// <summary>
-        /// <para>
-        /// Appends a frame representing an <see cref="Action{UIEventArgs}"/>-valued attribute.
-        /// </para>
-        /// <para>
-        /// The attribute is associated with the most recently added element. If the value is <c>null</c> and the
-        /// current element is not a component, the frame will be omitted.
-        /// </para>
-        /// </summary>
-        /// <param name="sequence">An integer that represents the position of the instruction in the source code.</param>
-        /// <param name="name">The name of the attribute.</param>
-        /// <param name="value">The value of the attribute.</param>
-        public void AddAttribute(int sequence, string name, Action<UIEventArgs> value)
```

Comment:
These are unused except for some unit tests. It's still possible to attach arbitrary delegates to the RTB - these methods are here to support method-group-to-delegate conversion and aren't really needed anymore.

Reply:
> It's still possible to attach arbitrary delegates to the RTB

But there's no use case for it, is there? Is this what we're discussing above with "should we remove the `Action`/`Func<T>` overloads of `AddAttribute`, or is this something else?

Reply:
> But there's no use case for it, is there?

The only use case for it would be making it nicer for someone writing RTB code by hand. Let's look at this again during the RTB API review.